src/pages/dashboard/components/menu/menu.style.tsx

```diff
@@ -1,7 +1,53 @@
 import styled from '@emotion/styled'
 
 export const MenuStyled = styled('nav')`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
   padding: 10px;
+  background: #ffffff;
+  width: 296px;
+`
+
+export const Logo = styled('div')`
+  margin-bottom: 60px;
+  display: flow;
+`
+
+export const MenuList = styled('ul')`
+  list-style: none;
+  padding: 0;
+  width: 100%;
+  display: compact;
+`
+
+export const MenuItem = styled('li')`
+  font-family: Poppins; // to nie jest generyk
+  width: 80%;
+  padding: 10px;
+  cursor: revert;
+  display: flex;
+  align-items: center;
+  font-size: 1.2em;
+
+  &:hover {
+    background-color: #9ac4fc;
```

Comment:
this is not the color.
colors should be extracted to variables. 

![image](https://github.com/rrasta/dashboard/assets/7860124/dc9c619a-3c40-4826-a4b8-88cff1d44358)